### ISSUE TITLE
When I re-launch minecraft, forge does not load.

### ISSUE THREAD

**Issue by Zlatex** [Pierce01/MinecraftLauncher-core#59]
When I re-launch minecraft, forge does not load.
I am using vue-cli-plugin-electron-builder.
If I restart the application the forge is loaded.
MCLC v. 3.16.1
forge v. 14.23.5.2854

**Comment by Zlatex** [comment]
```js
import { Client, Authenticator } from 'minecraft-launcher-core';
var launcher;
launcher = new Client();
        
launcher.on('data',(d)=>{
    console.log(d);
})
launcher.on('debug',(d)=>{
    console.log(d);
})
let opts = {
    clientPackage: null,
    root: "path",
    authorization: Authenticator.getAuth("username"),
    version: {
        number: "1.12.2",
        type: "release"
    },
    forge: "path/to/forge",
    memory: {
        max: "5000",
        min: "1000"
    }
} 
function launch(){
    launcher.launch(opts);
}
```
When I run the launch function again, then forge is not loaded. Only if I run it for the first time or restart the application.

**Comment by Pierce01** [comment]
Thats weird.. I'm unable to repro this. Is your launcher uploaded anywhere? I'll try looking into this when I can.

**Comment by Zlatex** [comment]
https://github.com/Zlatex/tmp
src/electron/components/startMinecraft.js

**Comment by Pierce01** [comment]
Sorry for taking so long, your issue has been fixed in the following commit: 449e15e
